webrtc.html

```diff
@@ -3284,39 +3284,59 @@ <h2>Attributes</h2>
+                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
               </dd>
               <dt><dfn>offerToReceiveVideo</dfn> of type <span class="idlMemberType"><a>boolean</a></span></dt>
               <dd>
-                <p>
-                  Whenever this is given a non-false value, and the
-                  <code><a>RTCPeerConnection</a></code> has no non-stopped
-                  "sendrecv" or "recvonly" video transceivers, createOffer()
-                  MUST as its first step invoke the equivalent of
-                  <code>addTransceiver("video")</code> on the
-                  <code><a>RTCPeerConnection</a></code> object, except that this
-                  MUST NOT <a>Update the negotiation-needed flag</a>,  and,
-                  provided this does not fail, proceed with createOffer()'s
-                  regular steps.
-                </p>
-                <p>
-                  In all other situations, it will be disregarded.
-                </p>
+                <p>Offer to receive video even though no video will be sent.
+                When this dictionary member is present as an option to <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a>, run the
+                <a>offer to receive</a> steps with a <var>mediaType</var> of
+                "video" instead of the regular <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
               </dd>
             </dl>
+            <p>To <dfn>offer to receive</dfn> with <var>mediaType</var>, run
+            the following steps:</p>
+            <ol>
+              <li>
+                <p>Let <var>connection</var> be the current <code>
+                <a>RTCPeerConnection</a></code> object.</p>
+              </li>
+              <li>
+                <p>If the value of the dictionary member is false, jump to the
+                step labeled <em>create offer</em>.</p>
+              </li>
+              <li>
+                <p>If <var>connection</var> has any stopped "sendrecv" or
+                "recvonly" transceivers of type <var>mediaType</var>, jump to
+                the step labeled <em>create offer</em>.</p>
+              </li>
+              <li>
+                <p>Let <var>transceiver</var> be the result of invoking the
+                equivalent of <code>connection.addTransceiver(mediaType)</code>,
+                except that this operation MUST NOT <a>update the
+                negotiation-needed flag</a>.</p>
+              </li>
+              <li>
+                <p>If <var>transceiver</var> is unset because the previous
+                operation threw an error, abort these steps.</p>
+              </li>
+              <li>
+                <p>Set <var>transceiver</var>'s <a>[[\Direction]]</a> slot to
+                "recvonly".</p>
+              </li>
+              <li>
+                <p><em>create offer:</em> Run the steps specified by <a href=
```

Comment:
I am a bit late and probably nitpicking but... when calling `createOffer({offerToReceiveAudio: true, offerToReceiveVideo: true})` won't this only process offerToReceiveAudio before running the normal createOffer steps?

Reply:
True.

Reply:
Proposed fix #1686 for this.